ui/page/governance/governance_page.go

```diff
@@ -59,15 +61,11 @@ func NewGovernancePage(l *load.Load) *Page {
 func (pg *Page) OnNavigatedTo() {
 	if activeTab := pg.CurrentPage(); activeTab != nil {
 		activeTab.OnNavigatedTo()
-	} else if pg.isGovernanceFeatureEnabled() {
+	} else {
 		pg.Display(NewProposalsPage(pg.Load))
 	}
 }
 
-func (pg *Page) isGovernanceFeatureEnabled() bool {
-	return pg.WL.SelectedWallet.Wallet.ReadBoolConfigValueForKey(sharedW.FetchProposalConfigKey, false)
-}
-
```

Comment:
proposals are not being fetched, and it's probably because we removed the config for fetching proposals

![Screenshot from 2023-10-18 11-13-41](https://github.com/crypto-power/cryptopower/assets/25265396/f25dce69-3bff-46dd-8225-1c915e7556de)